package.json

```diff
@@ -25,7 +25,6 @@
   },
   "author": "Erik Marks <rekmarks@protonmail.com>",
   "license": "ISC",
-  "private": false,
```

Comment:
Yarn seems to have made these changes to the manifest automatically (aside from `packageManager`).